sql/src/main/java/org/apache/druid/sql/calcite/planner/Rules.java

```diff
@@ -130,7 +130,6 @@
           ReduceExpressionsRule.FILTER_INSTANCE,
           ReduceExpressionsRule.CALC_INSTANCE,
           ReduceExpressionsRule.WINDOW_INSTANCE,
-          ReduceExpressionsRule.JOIN_INSTANCE,
```

Comment:
What are your thoughts on having a list of rules that are excluded in the codebase somewhere (maybe just commented out in here). That way we can add documentation in the code about why each rule was excluded.

If any of the reasoning for why a rule was removed from this list changes, it would be easy to add back the rules to transform to more efficient queries.

Reply:
Done. Added comments.